src/scheduler/CMakeLists.txt

```diff
@@ -13,4 +13,4 @@ set(LIB_FILES
 
 faabric_lib(scheduler "${LIB_FILES}")
 
-target_link_libraries(scheduler faabricmpi redis)
+target_link_libraries(scheduler state faabricmpi redis)
```

Comment:
This dependency is now needed to allow the top-level function server to flush state